lib/src/frame.dart

```diff
@@ -17,12 +17,36 @@ final _vmFrame = RegExp(r'^#\d+\s+(\S.*) \((.+?)((?::\d+){0,2})\)$');
 //     at VW.call$0 (eval as fn
 //         (https://example.com/stuff.dart.js:560:28), efn:3:28)
 //     at https://example.com/stuff.dart.js:560:28
-final _v8Frame =
+final _v8JsFrame =
     RegExp(r'^\s*at (?:(\S.*?)(?: \[as [^\]]+\])? \((.*)\)|(.*))$');
 
 // https://example.com/stuff.dart.js:560:28
 // https://example.com/stuff.dart.js:560
-final _v8UrlLocation = RegExp(r'^(.*?):(\d+)(?::(\d+))?$|native$');
+//
+// Group 1: URI, required
+// Group 2: line number, required
+// Group 3: column number, optional
```

Comment:
(If you are changing things anyway, consider using named capture groups here too, for consistency. Or not, can be done at any later time too.)

Reply:
Let's do unrelated changes separately. It helps with debugging, bisecting and reverting.